packages/@uppy/aws-s3/src/index.js

```diff
@@ -10,9 +10,21 @@ function resolveUrl (origin, link) {
   return new URL_(link, origin).toString()
 }
 
-function isXml (xhr) {
-  const contentType = xhr.headers ? xhr.headers['content-type'] : xhr.getResponseHeader('Content-Type')
-  return typeof contentType === 'string' && contentType.toLowerCase() === 'application/xml'
+function isXml (content, xhr) {
+  const contentType = (xhr.headers ? xhr.headers['content-type'] : xhr.getResponseHeader('Content-Type'))
+    // Get rid of mime parameters like charset=utf-8
+    .replace(/;.*$/, '')
+  if (typeof contentType === 'string') {
+    if (contentType.toLowerCase() === 'application/xml') {
```

Comment:
should we also be checking for `text/xml`? so maybe this?
```suggestion
    if (['application/xml', 'text/xml'].includes(contentType.toLowerCase())) {
```

Reply:
Warning: `includes` not supported in IE11, so probably `indexOf` solution is better?